README.md

```diff
@@ -99,6 +99,26 @@ If the toolbar `<script>` is accidentally included on your site, and `SentryTool
 
 #### Conditionally inserting script tag
 
-It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are elegible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
+It's possible to dynamically insert the script tag inside an SPA app, prior to calling `SentryToolbar.init()`, so that only users who are eligible . See docs/conditional-script.md for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login
 
 This example code will be eventually implemented as an NPM package, but for now it's something to be done manually.
+
+### Injecting the toolbar into a site
+
+To inject the toolbar into 'random' sites, use this snippet:
+
+Configure and paste the following into the console:
+
+1.
+```javascript
+var s = document.createElement('script');
+s.src = "https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js";
+document.head.appendChild(s);
+```
+2.
+```javascript
+window.SentryToolbar.init({ ... });
+```
+
+To view feature flags, you would need to set your feature flag configuration before you initialize the toolbar.
```

Comment:
```suggestion
To view feature flags, you will need to set your feature flag configuration before you initialize the toolbar.
```
should we include a link to where they can learn how to configure it?

Reply:
the docs for how to configure this are not out yet :( but yes once they're out we should!